bundles/io/org.eclipse.smarthome.io.transport.serial.api/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,19 @@
+Manifest-Version: 1.0
+Bundle-ActivationPolicy: lazy
+Bundle-ClassPath: .
+Bundle-ManifestVersion: 2
+Bundle-Name: Eclipse SmartHome Serial Transport API
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-SymbolicName: org.eclipse.smarthome.io.transport.serial.api;singleton:=true

```

Comment:
I would prefer `org.eclipse.smarthome.io.transport.serial` as a bundle + package name - we usually never use `api` in such situations.